### ISSUE TITLE
Integrate scale/rotation of assets from frontend

### ISSUE THREAD

**Issue by nicolocarpignoli** [AR-js-org/studio-backend#27]
If frontend side calls the methods to generate templates with scale/rotation proprties, on the backend side we should handle them and translate them into a-frame properties, on the template.

**Comment by le0m** [comment]
Frontend calls `new Package({...})` with a new parameter `assetParam`, structured like this:

```
{
  isValid: <bool>,
  scale: <float>,
  size: {
    width: <float>,
    height: <float>,
    depth: <float>
  }
}
```

**note**: check that `isValid === true` before using!

**Comment by le0m** [comment]
solved with #34
